cluster/src/main/java/com/hartwig/pipeline/Arguments.java

```diff
@@ -17,6 +17,120 @@
 
     boolean cleanup();
 
+    String DEFAULT_DOCKER_CMEK = "/secrets/storage-cmek.json";
```

Comment:
Normal the CMEK is actually just the name of the key used ie. projects/project/location/blah/blah. Does this actually work having it a json file? I don't see why running in Docker would need to impact CMEK (it's more like the prokect or location setttings).